app/services/dwp/benefit_check_service.rb

```diff
@@ -4,10 +4,6 @@ class BenefitCheckService
     USE_MOCK = ActiveModel::Type::Boolean.new.cast(ENV.fetch('BC_USE_DEV_MOCK', nil))
     REQUEST_TIMEOUT = 30.seconds
 
-    class ApiError < StandardError
```

Comment:
I would say, even if get rid of Nesty, we still want to have a custom exception class to raise when rescuing from `rescue Savon::SOAPFault`. Maybe just remove nesty, leaving the rest of the code?